src/index.js

```diff
@@ -275,7 +276,7 @@ export const componentClasses = [
  */
 const init = () => {
   if (!settings.disableAutoInit) {
-    initCheckbox();
+    //initCheckbox();
```

Comment:
Let’s not forget to remove this like before finishing up the work 😉

Reply:
👍